src/api/getTokenHolders.js

```diff
@@ -0,0 +1,11 @@
+const { get } = require('request-promise')
+
+module.exports = async (dvf, token) => {
+  if (token) {
```

Comment:
I think this could be simplified by just returning the result of the request. The null check for token can be omitted, or should throw an error instead (since it doesn't really make sense to call this function without it)